src/foraging_gui/GenerateMetadata.py

```diff
@@ -1017,7 +1123,12 @@ def _get_behavior_software(self):
             version=f'behavior branch:{self.Obj["current_branch"]}   commit ID:{self.Obj["commit_ID"]}    version:{self.Obj["version"]}; metadata branch: {current_branch}   commit ID:{commit_ID}   version:{version}',
             url=self.Obj["repo_url"],
         ))
```

Comment:
Whats going on here? It seems like we are adding Software info to "behavior_software" and "behavior_software_origin"

Reply:
The behavior_software contains both the software version for running the behavior and the software version for generating the session metadata. The behavior_software_origin only contains the version for running the behavior. I have removed the behavior_software_origin for simplicity.